lib/pathfinder.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+const Errors = require('./errors')
+
+const lifecycleStages = [
+  'configure',
```

Comment:
No need the `validate` callback here ? Don't know if it can be useful but it doesn't cost much to put it right ?